exp/anbox-cloud.md

```diff
@@ -77,6 +80,42 @@ The main components powering the streaming stack in Anbox Cloud are:
 
 **NATS**: A messaging system that the different components use to communicate (see the [project page](https://github.com/nats-io)).
 
+### LXD
+
+Anbox Cloud includes LXD for hosting and managing the Ubuntu containers that run the Android containers.
+
+LXD is installed through the `ams-lxd` charm, which adds some Anbox-specific configuration to LXD. AMS configures LXD automatically and fully manages it, which means that in most scenarios, you do not need to worry about LXD at all.
+
+If you want to check what LXD is doing, you can always run `lxc list` to display the existing containers. For the full deployment, LXD is used to host the AMS containers. If you run the Anbox Cloud Appliance, LXD is in addition used to containerise the Anbox Cloud deployment, which means that it hosts containers for the different Juju machines that Anbox Cloud requires:
+
+```
++--------------------------+---------+------------------------+------+-----------+-----------+----------+
+|           NAME           |  STATE  |          IPV4          | IPV6 |   TYPE    | SNAPSHOTS | LOCATION |
++--------------------------+---------+------------------------+------+-----------+-----------+----------+
+| ams-cdr3292j841mb72gksog | RUNNING | 192.168.96.1 (eth0)    |      | CONTAINER | 0         | lxd0     |
+|                          |         | 192.168.250.1 (anbox0) |      |           |           |          |
++--------------------------+---------+------------------------+------+-----------+-----------+----------+
+| juju-73e292-0            | RUNNING | 240.0.233.254 (eth0)   |      | CONTAINER | 0         | lxd0     |
++--------------------------+---------+------------------------+------+-----------+-----------+----------+
+| juju-73e292-1            | RUNNING | 240.0.98.38 (eth0)     |      | CONTAINER | 0         | lxd0     |
++--------------------------+---------+------------------------+------+-----------+-----------+----------+
+| juju-73e292-2            | RUNNING | 240.0.61.208 (eth0)    |      | CONTAINER | 0         | lxd0     |
++--------------------------+---------+------------------------+------+-----------+-----------+----------+
+| juju-34631c-0            | RUNNING | 240.0.180.30 (eth0)    |      | CONTAINER | 0         | lxd0     |
```

Comment:
To we need to change/hide the IP addresses here?

Reply:
No, these are fine to keep